froide/comments/templates/froide_comments/form.html

```diff
@@ -22,10 +22,26 @@
 
   {% render_field form.comment %}
 
+  {% is_moderator as is_moderator %}
+  {% if is_moderator and not request.user == object.request.user %}
+  <div class="mb-3 row">
+    <div class="col-md-4 fw-bold">
+      {% trans "Moderator Badge" %}
+    </div>
+    <div class="col-md-8">
+      <div class="form-check">
```

Comment:
You can use the `{% render_field form.field_name %}` template tag from the `form_helper` template tag library.

Reply:
Also I just discovered that our comment forms don't use prefixes and ids clash because the form is embedded multiple times on the page. So labels are not properly linked to their fields. 😭

Reply:
Will try again